testutils/vcamera/cmd/main.go

```diff
@@ -0,0 +1,40 @@
+//go:build linux
+
+// This package creates two virtual cameras and streams test video to them until the program
+// halts (with ctrl-c for example). These cameras are discoverable by app.
+//
+// Prerequisites: run etc/v4l2loopback_setup.sh which will install GStreamer and V4L2Loopback.
+package main
+
+import (
+	"github.com/edaniels/golog"
+	"go.viam.com/rdk/testutils/vcamera"
```

Comment:
@edaniels For some reason our linter likes to remove these imports after running `make lint-go`. Have you seen that before or know why that would be? Wanted to ask before I dive any deeper into the problem.